docs/_legacy/Gas-Price-Oracle.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Gas price oracle
```

Comment:
OK, and this one

Reply:
Note this collection is hidden in the sidebar.